collections/_events/2023-10-26-summit-nescala.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: event
+
+title: "Typelevel Summit NEScala"
+short_title: "Typelevel Summit"
+date_string: "October 26, 2023"
+location: "Online"
+description: "One day of recorded talks and one day of unconference, co-located with NE Scala."
+
+---
+
+## About the Summit
+
+The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of recorded talks and one day of (shared) unconference.
```

Comment:
```suggestion
The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of talks and socializing online and one day of unconference.  Talks will be recorded and available online afterwards.
```

Reply:
maybe "discussion and socializing"?

Reply:
or "one of day of talks and one day of unconference, with discussion and socializing online throughout"? idk

Reply:
I like that second version!